package.json

```diff
@@ -32,6 +33,17 @@
     "eslint": "^7.22.0",
     "jstat": "^1.9.4",
     "mocha": "^8.3.2",
+    "nyc": "^15.1.0",
+    "source-map-support": "^0.5.19",
     "ts-node": "^9.1.1"
+  },
+  "nyc": {
+    "extension": [
+      ".ts"
+    ],
+    "exclude": [
+      "**/*.d.ts"
+    ],
+    "all": true
```

Comment:
```suggestion

```
I think you should remove the `all` property here because otherwise every time we use the `nyc` it will include all files and as you know there are some situations (the one I am working on) that should not run for all files. Alternatively, we can use the `--all` in the coverage command above.